### ISSUE TITLE
attachedWidget is not repositioned

### ISSUE THREAD

**Issue by deepak786** [MMMzq/bot_toast#73]
```
BotToast.showAttachedWidget(
      attachedBuilder: (_) => Toast(message),
      duration: Duration(seconds: 2),
      targetContext: targetContext,
);
```

1. The keyboard is visible.
2. Click on the button to show the attached toast with the above code.
3. close the keyboard.
4. The toast is not repositioned as the widget to which it attached is now at a different position.

**Comment by MMMzq** [comment]
Do you think it is ok to automatically close Toast when the keyboard pops up?

**Comment by deepak786** [comment]
> Do you think it is ok to automatically close Toast when the keyboard pops up?

Can be but depends on the use case.

**Comment by MMMzq** [comment]
I think this is necessary, otherwise there will be bug.

**Comment by MMMzq** [comment]
Fix:`v3.0.2`

**Comment by deepak786** [comment]
I will test the version 3.0.2 and will share the feedback accordingly.
